src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx

```diff
@@ -45,6 +45,20 @@ const StyledGelWrapperGrid = styled(GelPageGrid)`
   }
 `;
 
+const StyledGridItemParagraph = styled(Grid)`
+  @media (min-width: 22.5625rem) and (max-width: 37.4375rem) {
```

Comment:
Oh the pains caused by making a grid element disappear at an irregular breakpoint.

I don't think we re-assign these grid values anywhere else in the codebase - we'd always pass the groups object into the grids' `columns`  prop. I realise this isn't possible here because those 'groups' are assigned to our gel breakpoints, which don't conform to the change being made.

Not sure the best approach here. I guess it's either - change the designs to match our existing breakpoints (i.e. essentially return this to as it is already on live), or accept these one-off grid restyles.
 
There might be another option though - it would be good to get the other engineers' thoughts.

Also regardless of approach - `GEL_GROUP_2_SCREEN_WIDTH_MAX` can be used instead of `37.4375rem`